orttraining/orttraining/python/training/_ortmodule_io.py

```diff
@@ -18,9 +64,49 @@ def deepcopy_model_input(*inputs, **kwargs):
 
     return sample_inputs_copy, sample_kwargs_copy
 
-class _TensorStub:
-    # Stub for a torch.Tensor value to be used to formulate the output schema
-    pass
+
+class _TensorStub(object):
+    '''Tensor stub class used to represent model's input or output'''
+
+    def __init__(self, name=None, dtype=None, shape=None, shape_dims=None):
+        self.name = name
```

Comment:
Shall we make all these attributes internal using the _ prefix?

Reply:
This is already an internal data-class, never seen/used by users. Attributes must start with `_` when they are private and shouldn't be directly accessed